llvm-build/utils.ps1

```diff
@@ -0,0 +1,278 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+if (Test-Path function:\exec) {
+    # already included
+    return
+}
+
+if (!(Test-Path function:\Get-RepoRoot)) {
+    # git revparse uses cwd. E2E builds use a different
+    # working dir, so we pin it to out repo (submodule in E2E)
+    function Get-RepoRoot {
+        exec -wd $PSScriptRoot {
+            git rev-parse --show-toplevel
+        }
+    }
+}
```

Comment:
I have an impression that `exec` is not defined by the moment of line 13.
Some time later someone can add a change that calls `Get-RepoRoot` (line 12) earlier than `exec` is defined (line 55), and get something unexpected.

Reply:
Actually, between this and the comment below, I think I need to review the whole utils file and prune it. I don't think some of these are used anywhere...